docs/application/native/guides/app-management/widget-viewer.md

```diff
@@ -104,31 +104,33 @@ After displaying a widget, you can manage the widget application's lifecycle:
 
 After displaying widget, you can retrieve information about widget instances as shown in the following code.
 
-1. Get the content information using `widget_viewer_evas_get_content_info()`. The parameter of `widget_viewer_evas_get_content_info()` is a widget object created by widget_viewer_evas_add_widget:
+1. Create widget object with `widget_viewer_evas_add_widget()`.
 
    ```
    Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
+   ```
+
+2. Get the content information using `widget_viewer_evas_get_content_info()` with the widget object as the parameter.:
+
+   ```
    const char *content = widget_viewer_evas_get_content_info(widget);
    ```
 
-2. Get widget ID from the widget object using `widget_viewer_evas_get_widget_id()`. The parameter for `widget_viewer_evas_get_widget_id()` is a widget object created by `widget_viewer_evas_add_widget()`:
+3. Get widget ID from the widget object using `widget_viewer_evas_get_widget_id()` with the widget object as the parameter.:
 
    ```
-   Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
    const char *widget_id = widget_viewer_evas_get_widget_id(widget);
    ```
 
-3. Get widget update period from the widget object using an `widget_viewer_evas_get_period()` function. Its parameter is an widget object created by the `widget_viewer_evas_add_widget` function.
+4. Get widget update period from the widget object using an `widget_viewer_evas_get_period()` with the widget object as the parameter.:
 
    ```
-   Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
    double period = widget_viewer_evas_get_period(widget);
    ```
 
-4. Get widget instance id from the widget object using an `widget_viewer_evas_get_widget_instance_id()` function. Its parameter is an widget object created by the `widget_viewer_evas_add_widget` function.
+5. Get widget instance id from the widget object using an `widget_viewer_evas_get_widget_instance_id()` with the widget object as the parameter.:
```

Comment:
Please remove "an" before `widget_viewer_evas_get_widget_instance_id()`.
Consider the same for line 125 also.